_2020/data-wrangling.md

```diff
@@ -111,7 +116,7 @@ you can pass `-E`.
 So, looking back at `/.*Disconnected from /`, we see that it matches
 any text that starts with any number of characters, followed by the
 literal string "Disconnected from &rdquo;. Which is what we wanted. But
-beware, regular expressions are trixy. What if someone tried to log in
+beware, regular expressions are tricky. What if someone tried to log in
```

Comment:
I used trixy intentionally :)

Reply:
Ah, ok. I've never seen that term before, and a quick search didn't reveal anything that seemed relevant. Full speed ahead.

Reply:
Oh, yeah, it's not a real word, more of an personal ism :)